Alembic related failures on Focal

## Description

We have various alembic related failures on Focal application tests. 

## Steps to Reproduce

- [ ] Create a fresh virtualenv from develop
- [ ] To test on focal, first move to `dev_focal` branch. `git checkout dev_focal`. 
- [ ] `make test-focal`  

## Expected Behavior

Tests should pass.


## Actual Behavior

```

FAILED tests/test_alembic.py::test_alembic_head_matches_db_models - AssertionError: Schema for ('table', 'journalist_login_attempt', 'journalist_login_attempt') did not match:
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[60f41bb14d98] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'f2833ac34bb6']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[523fff3f969c] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'f2833ac34bb6']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[3da3fcab826a] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'f2833ac34bb6']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[48a75abc0121] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'f2833ac34bb6']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[a9fe328b053a] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'f2833ac34bb6']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[35513370ba0d] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'f2833ac34bb6']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[b58139cfdc8c] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'f2833ac34bb6']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_alembic_migration_downgrade[f2833ac34bb6] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', 'e0a525cbab83']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_schema_unchanged_after_up_then_downgrade[60f41bb14d98] - AssertionError: Schema for ('table', 'revoked_tokens', 'revoked_tokens') did not match:
FAILED tests/test_alembic.py::test_schema_unchanged_after_up_then_downgrade[fccf57ceef02] - AssertionError: Schema for ('table', 'submissions', 'submissions') did not match:
FAILED tests/test_alembic.py::test_schema_unchanged_after_up_then_downgrade[3d91d6948753] - AssertionError: Schema for ('table', 'replies', 'replies') did not match:
FAILED tests/test_alembic.py::test_schema_unchanged_after_up_then_downgrade[e0a525cbab83] - AssertionError: Schema for ('table', 'replies', 'replies') did not match:
FAILED tests/test_alembic.py::test_schema_unchanged_after_up_then_downgrade[b58139cfdc8c] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', '-1']' returned non-zero exit status 1.
FAILED tests/test_alembic.py::test_schema_unchanged_after_up_then_downgrade[f2833ac34bb6] - AssertionError: Schema for ('table', 'journalist_login_attempt', 'journalist_login_attempt') did not match:
FAILED tests/test_alembic.py::test_schema_unchanged_after_up_then_downgrade[2d0ce3ee5bdc] - AssertionError: Schema for ('table', 'journalist_login_attempt', 'journalist_login_attempt') did not match:
FAILED tests/test_alembic.py::test_downgrade_with_data[b58139cfdc8c] - subprocess.CalledProcessError: Command '['alembic', 'downgrade', '-1']' returned non-zero exit status 1.
```

## Comments

Suggestions to fix, any other relevant information.

--- a/securedrop/alembic/versions/2d0ce3ee5bdc_added_passphrase_hash_column_to_.py
+++ b/securedrop/alembic/versions/2d0ce3ee5bdc_added_passphrase_hash_column_to_.py
@@ -23,7 +23,8 @@ def upgrade():
 def downgrade():
     # sqlite has no `drop column` command, so we recreate the original table
     # then load it from a temp table
-
+    conn = op.get_bind()
+    conn.execute("PRAGMA legacy_alter_table=ON")
     op.rename_table("journalists", "journalists_tmp")
 
     op.create_table(
@@ -43,7 +44,6 @@ def downgrade():
         sa.UniqueConstraint("username"),
     )
 
-    conn = op.get_bind()
     conn.execute(
         """
         INSERT INTO journalists

--- a/securedrop/alembic/versions/3d91d6948753_create_source_uuid_column.py
+++ b/securedrop/alembic/versions/3d91d6948753_create_source_uuid_column.py
@@ -18,6 +18,8 @@ depends_on = None
 
 
 def upgrade():
+    conn = op.get_bind()
+    conn.execute("PRAGMA legacy_alter_table=ON")
     # Schema migration
     op.rename_table("sources", "sources_tmp")
 
@@ -25,7 +27,6 @@ def upgrade():
     op.add_column("sources_tmp", sa.Column("uuid", sa.String(length=36)))
 
     # Add UUIDs to sources_tmp table.
-    conn = op.get_bind()
     sources = conn.execute(sa.text("SELECT * FROM sources_tmp")).fetchall()
 
     for source in sources:

--- a/securedrop/alembic/versions/60f41bb14d98_add_session_nonce_to_journalist.py
+++ b/securedrop/alembic/versions/60f41bb14d98_add_session_nonce_to_journalist.py
@@ -17,6 +17,8 @@ depends_on = None
 
 
 def upgrade():
+    conn = op.get_bind()
+    conn.execute("PRAGMA legacy_alter_table=ON")
     # Save existing journalist table.
     op.rename_table('journalists', 'journalists_tmp')
 
@@ -24,7 +26,6 @@ def upgrade():
     op.add_column('journalists_tmp', sa.Column('session_nonce', sa.Integer()))
 
     # Populate nonce column.
-    conn = op.get_bind()
     journalists = conn.execute(
         sa.text("SELECT * FROM journalists_tmp")).fetchall()
 
@@ -57,7 +58,6 @@ def upgrade():
         sa.UniqueConstraint('uuid')
     )
 
-    conn = op.get_bind()
     conn.execute('''
         INSERT INTO journalists
         SELECT id, uuid, username, first_name, last_name, pw_salt, pw_hash,

--- a/securedrop/alembic/versions/6db892e17271_add_reply_uuid.py
+++ b/securedrop/alembic/versions/6db892e17271_add_reply_uuid.py
@@ -18,6 +18,8 @@ depends_on = None
 
 
 def upgrade():
+    conn = op.get_bind()
+    conn.execute("PRAGMA legacy_alter_table=ON")
     # Schema migration
     op.rename_table("replies", "replies_tmp")
 
@@ -25,7 +27,6 @@ def upgrade():
     op.add_column("replies_tmp", sa.Column("uuid", sa.String(length=36)))
 
     # Populate new column in replies_tmp table.
-    conn = op.get_bind()
     replies = conn.execute(sa.text("SELECT * FROM replies_tmp")).fetchall()
 
     for reply in replies:

--- a/securedrop/alembic/versions/e0a525cbab83_add_column_to_track_source_deletion_of_.py
+++ b/securedrop/alembic/versions/e0a525cbab83_add_column_to_track_source_deletion_of_.py
@@ -17,6 +17,8 @@ depends_on = None
 
 
 def upgrade():
+    conn = op.get_bind()
+    conn.execute("PRAGMA legacy_alter_table=ON")
     # Schema migration
     op.rename_table("replies", "replies_tmp")
 
@@ -24,7 +26,6 @@ def upgrade():
     op.add_column("replies_tmp", sa.Column("deleted_by_source", sa.Boolean()))
 
     # Populate deleted_by_source column in replies_tmp table.
-    conn = op.get_bind()
     replies = conn.execute(sa.text("SELECT * FROM replies_tmp")).fetchall()
 
     for reply in replies:

--- a/securedrop/alembic/versions/f2833ac34bb6_add_uuid_column_for_users_table.py
+++ b/securedrop/alembic/versions/f2833ac34bb6_add_uuid_column_for_users_table.py
@@ -18,6 +18,8 @@ depends_on = None
 
 
 def upgrade():
+    conn = op.get_bind()
+    conn.execute("PRAGMA legacy_alter_table=ON")
     # Save existing journalist table.
     op.rename_table("journalists", "journalists_tmp")
 
@@ -25,7 +27,6 @@ def upgrade():
     op.add_column("journalists_tmp", sa.Column("uuid", sa.String(length=36)))
 
     # Add UUIDs to journalists_tmp table.
-    conn = op.get_bind()
     journalists = conn.execute(sa.text("SELECT * FROM journalists_tmp")).fetchall()
 
     for journalist in journalists:
@@ -57,7 +58,6 @@ def upgrade():
         sa.UniqueConstraint("uuid"),
     )
 
-    conn = op.get_bind()
     conn.execute(
         """
         INSERT INTO journalists

--- a/securedrop/alembic/versions/fccf57ceef02_create_submission_uuid_column.py
+++ b/securedrop/alembic/versions/fccf57ceef02_create_submission_uuid_column.py
@@ -18,6 +18,8 @@ depends_on = None
 
 
 def upgrade():
+    conn = op.get_bind()
+    conn.execute("PRAGMA legacy_alter_table=ON")
     # Schema migration
     op.rename_table("submissions", "submissions_tmp")
 
@@ -25,7 +27,6 @@ def upgrade():
     op.add_column("submissions_tmp", sa.Column("uuid", sa.String(length=36)))
 
     # Add UUIDs to submissions_tmp table.
-    conn = op.get_bind()
     submissions = conn.execute(sa.text("SELECT * FROM submissions_tmp")).fetchall()
 
     for submission in submissions:

